test/cluster/auth_cluster/test_raft_service_levels.py

```diff
@@ -114,6 +114,67 @@ async def test_service_levels_upgrade(request, manager: ManagerClient, build_mod
     result_with_sl_v2 = await cql.run_async(f"SELECT service_level FROM system.service_levels_v2")
     assert set([sl.service_level for sl in result_with_sl_v2]) == set(sls + [DRIVER_SL_NAME] + [sl_v2])
 
+async def validate_connections_scheduling_group(cql, hosts, username, scheduling_group):
```

Comment:
Andrzej already introduced a way to test correct sgs via metrics: _verify_tasks_processed_metrics

Perhaps we don't need another way?

To me looks like connection checking is fragile